draft-ietf-tls-keylogfile.md

```diff
@@ -203,6 +219,27 @@ additional information.
 An implementation of TLS 1.2 {{!TLS12}} (and also earlier versions) use the
 label "CLIENT_RANDOM" to identify the "master" secret for the connection.
 
+## Secret Labels for ECH
+
+With the introduction of ECH {{!ECH}} additional secrets are derived
+during the handshake to encrypt the Inner ClientHello message using Hybrid Public
+Key Encryption (HPKE) {{?RFC9180}}. The client SHOULD log the ECH labels described below
+if it offered ECH regardless of server acceptance. The server MAY log the labels only if it
+successfully decrypted and accepted ECH offered by the client. The client MUST NOT log
+the labels for connections that use the GREASE ECH extension (see Section 6.2 of {{!ECH}}).
```

Comment:
```suggestion
```

Reply:
Not sure if I follow this. I thought asking implementors not to pollute SSLKEYLOGFILE with imaginary artefacts from grease is a good thing, no?

Reply:
There isn't a secret or config to log when greasing, so why would it be logged?

Reply:
I've checked few implementations and indeed they are generating GREASE without creating any ephemeral secrets. Which makes perfect sense of course. Thanks for pointing this out!